### PR TITLE
Add protocol for codenames

### PR DESCRIPTION
**Pull request by 2quarius**

### CODE REVIEW

File: codenames/codenames.proto [hunk 1]
```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package Codenames;
+
+message StartGameSettings {
+    bool isLessThan4Users = 1;
```
Comment (Veiasai) [inline codenames/codenames.proto]: Couldn't you get this by counting the user?...

Comment (2quarius) [reply codenames/codenames.proto]: Could. But if user is less than 4 the rules of the game changes. And I haven't decide when to implement it, just leave it be a setting.

**Review by Veiasai — APPROVED** [review]

**Review by Veiasai — CHANGES_REQUESTED** [review]
changelog
https://github.com/SailGame/proto/blob/main/changelog.md
something like  v1.1.0  description.

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
 ## 1.0.0
 
-[Cookbook-for-Provider-Dev](https://github.com/SailGame/Core/wiki/Cookbook-for-Provider-Dev)
\ No newline at end of file
+[Cookbook-for-Provider-Dev](https://github.com/SailGame/Core/wiki/Cookbook-for-Provider-Dev)
+
+### Codenames
```
Comment (Veiasai) [inline changelog.md]:
```
## 1.1.0
- CodeNames
- xxx
-
## 1.0.0
```

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+## 1.1.0
+- [Codenames](https://github.com/SailGame/Codenames/wiki/Design-Report)
```
Comment (Veiasai) [inline changelog.md:2]: I don't mind just leave the content here,  but a link is fine. 
good job.

**Review by Veiasai — APPROVED** [review]

**Pull request merged**